__tests__/api.test.js

```diff
@@ -271,6 +271,37 @@ describe("/api/articles", () => {
                     });
             });
         });
+        describe("filtering queries", () => {
```

Comment:
Also worth throwing in a test for a topic that does exist but has no articles, the go-to for this in the test data is 'paper'